src/functions/module_array.ts

```diff
@@ -66,6 +66,22 @@ export const EXPAND: AddFunctionDescription = {
   isExported: true,
 };
 
+// -----------------------------------------------------------------------------
+// FLATTEN
+// -----------------------------------------------------------------------------
+export const FLATTEN: AddFunctionDescription = {
+  description: _lt("Flattens all the values from one or more ranges into a single column."),
+  args: [
+    arg("range (range<any>)", _lt("The first range to flatten.")),
+    arg("range2 (range<any>, repeating)", _lt("Additional ranges to flatten.")),
```

Comment:
range and range2 could be simple value: ```any, range<any>```

maybe add test with simple values
the code will probably need to be modified to consider this case